base/multidimensional.jl

```diff
@@ -1466,7 +1466,14 @@ end
 
 ## Permute array dims ##
 
-function permutedims(B::StridedArray, perm)
+"""
+    permutedims(v::AbstractVector)
+
+Reshapes vector `v` into a `1 × length(v)` row matrix.
```

Comment:
Typically we stick to the imperative in docstrings, so this should be "reshape" instead of "reshapes."

Reply:
Thanks - I always forget this!